cypress/fixtures/document_submissions/get-many.json

```diff
@@ -63,12 +66,14 @@
     "documentType": {
       "id": "proof-of-id",
       "title": "Proof of ID",
-      "description": "A valid document that can be used to prove identity"
+      "description": "A valid document that can be used to prove identity",
+      "enabled": true
     },
     "staffSelectedDocumentType": {
-      "id": "proof-of-id",
-      "title": "Proof of ID",
-      "description": "Any valid proof of id"
+      "id": "passport-scan",
+      "title": "Passport",
```

Comment:
Updated this to make sure that the staff selected doc type was different to the doc type on the page to make it more true to life